src/main/java/org/opensearch/knn/index/query/rescore/RescoreContext.java

```diff
@@ -43,30 +43,6 @@ public final class RescoreContext {
      * Flag to track whether the oversample factor is user-provided or default. The Reason to introduce
      * this is to set default when Shard Level rescoring is false,
      * else we end up overriding user provided value in NativeEngineKnnVectorQuery
-     *
-     *
-     * This flag is crucial to differentiate between user-defined oversample factors and system-assigned
```

Comment:
Why is this invalid? Can we just add <p> if necessary?

Reply:
Yeah ..I thought it's long ,..,and some of them causing Java Docs to Fail , so thought of keep it simple ..Let me add back!!